src/main/java/coydir/model/person/Leave.java

```diff
@@ -0,0 +1,61 @@
+package coydir.model.person;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Leave in the database.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidLeave}
+ */
+public class Leave {
+    private static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+    public final LocalDate startDate;
+    public final LocalDate endDate;
+
+    /**
+     * Constructs a {@code Leave}.
+     *
+     * @param startDate A valid start date.
+     * @param endDate A valid start date.
+     */
+    public Leave(String startDate, String endDate) {
+        this.startDate = LocalDate.parse(startDate, FORMAT);
+        this.endDate = LocalDate.parse(endDate, FORMAT);
+    }
+
+    /**
+     * Returns true if a endDate and StartDate makes sense.
+     */
+    public static boolean isValidLeave(String startDate, String endDate) {
+        final int b = LocalDate.parse(endDate, FORMAT).compareTo(LocalDate.parse(startDate, FORMAT));
+        return b > 0;
+    }
+
+    /**
+     * Returns number of days
+     */
+    public int getTotalDays() {
+        return this.endDate.compareTo(this.startDate);
```

Comment:
Eh wait does `compareTo` return the number of days? If so that kinda explains the previous one.

Reply:
Based on my experimenting, `compareTo` actually returns the number of days inbetween